### PR TITLE
Added get_latest_model method

### PR DESCRIPTION
**Pull request by starlord-daniel**
The get_latest_model method helps to get the latest model without providing a specific tag.

### CODE REVIEW

**Comment by dtzar** [comment]
I don't like having helper code which isn't used anywhere in the rest of the solution....  Since this is a heavy dupe of the method with the tag, I'd rather mod the same method to accept a tag or not and return the latest model.

**Comment by starlord-daniel** [comment]
@dtzar Makes sense, I've adapted the code accordingly.

File: diabetes_regression/util/model_helper.py [hunk 1]
```diff
@@ -44,37 +44,44 @@ def get_model_by_tag(
...
-        model_list = AMLModel.list(
-            aml_workspace, name=model_name,
-            tags=[[tag_name, tag_value]], latest=True
-        )
+        model_list = None
+        tag_ext = ""
+
+        # Get lastest model
+        # True: by name and tags
+        if tag_name is not None or tag_value is not None:
```
Comment (eedorenko) [inline diabetes_regression/util/model_helper.py]: I believe both of them should be not None.
Or at least tag_name.

Comment (starlord-daniel) [reply diabetes_regression/util/model_helper.py]: Updated accordingly.

**Pull request merged**